packages/react/src/date-range-picker/DateRangePicker.tsx

```diff
@@ -234,10 +241,22 @@ export const DateRangePicker = ({
 				return;
 			}
 
+			// Trigger the callbacks
 			onChange(nextValue);
 			onToInputChangeProp?.(inputValue);
 		},
-		[maxDate, minDate, onChange, onToInputChangeProp, valueAsDateOrUndefined]
+		250
```

Comment:
Should equal the rest
```suggestion
		200
```